.github/workflows/podvm_builder.yaml

```diff
@@ -0,0 +1,59 @@
+name: Create Pod VM Builder Image
+on:
+  release:
+    types: [created]
```

Comment:
maybe worth adding manual option? not critical ATM though

Reply:
We can add manual running of workflow in subsequent PRs